modules/dbparser/patterndb.c

```diff
@@ -699,12 +707,19 @@ _pattern_db_process(PatternDB *self, PDBLookupParams *lookup, GArray *dbg_list)
   return process_params->rule != NULL;
 }
 
+static void
+_prepare_lookup_params(PDBLookupParams *lookup, LogMessage *msg, LogTemplate *program_template)
+{
```

Comment:
shouldn't we just introduce program_template argument to pdb_lookup_params_init()? That would eliminate the need for this wrapper function, which is difficult to name in a way that actually tells what it does.

Reply:
You are right! I'll look into it.

Reply:
Fixed.